src/woo_publications/api/utils.py

```diff
@@ -0,0 +1,16 @@
+import re
+
+from djangorestframework_camel_case.util import (
+    camelize_re,
+    underscore_to_camel as _underscore_to_camel,
+)
+
+
+def underscore_to_camel[T: str | int](input_: T) -> T:
+    """
+    Convert a string from under_score to camelCase.
+    """
+    if not isinstance(input_, str):
+        return input_
```

Comment:
Tests indeed seem to not be picked up, since there are still missing coverage annotations.